src/App.test.js

```diff
@@ -3,6 +3,7 @@ import App from './App';
 
 test('renders learn react link', () => {
   render(<App />);
-  const linkElement = screen.getByText(/Math Magicians/i);
+  const linkElement = screen.getByText(/Magicians/i);
+
   expect(linkElement).toBeInTheDocument();
 });
```

Comment:
- [ ] `Required`
- [ ] Great job so far! 👏 
- [ ] However the Project requirements stated
- [ ] All the functionalities & scenarios of the operate.js and calculate.js modules should be covered using Jest.
- [ ] But you only wrote one test.
- [ ] Please write more tests as the requirements states 🥂